src/components/NetworkLabel/NetworkLabel.tsx

```diff
@@ -0,0 +1,43 @@
+import { ReactElement } from 'react'
+import { makeStyles } from '@material-ui/core/'
+
+import { getNetworkInfo } from 'src/config'
+import { NetworkInfo } from 'src/config/networks/network'
+import { border, extraSmallFontSize, sm, xs, fontColor } from 'src/theme/variables'
+
+type Props = {
+  networkInfo?: NetworkInfo
+  onClick?: () => void
+  flexGrow?: boolean
+}
+
+function NetworkLabel({ networkInfo, onClick, flexGrow }: Props): ReactElement {
+  const selectedNetwork = networkInfo || getNetworkInfo()
+  const backgroundColor = selectedNetwork.backgroundColor
+  const textColor = selectedNetwork.textColor
+  const classes = useStyles({ onClick, backgroundColor, textColor, flexGrow })
+
+  return (
+    <span className={classes.root} onClick={onClick}>
+      {selectedNetwork.label}
+    </span>
+  )
+}
+
+export default NetworkLabel
+
+const useStyles = makeStyles({
```

Comment:
`useStyles`/`makeStyles` is/will be legacy with the release of MUI v5. It's not necessary, but I would suggest doing this instead in `styled-components`. What do you think? I'll mark any other places this pops up in this PR too.